### DISCUSSION TITLE
"ALL" property for whitelisted-drops

### DISCUSSION THREAD

**Discussion by CayeOficial** [BG-Software-LLC/WildTools#140] [Pending Feature Requests] [answered]
Would be great to just have to put "ALL" or "*" on the whitelisted-drops section and it'll whitelist automatically all drops but not the specified on blacklisted-drops; that would save some effort and I think is a basic feature.

**Comment by OmerBenGera** [comment]
Remove the whitelisted-drops section, it has the same effect.

**Comment by OmerBenGera** [comment, marked as answer]
Remove the whitelisted-drops section, it has the same effect.
